packages/main/test/samples/TabContainer.sample.html

```diff
@@ -391,4 +391,75 @@ <h3>TabContainer with nested tabs</h3>
 	</xmp></pre>
 </section>
 
+<!-- Background Design -->
+
+<section>
+	<style>
+		.radio-button-group {
+			display: flex;
+			flex-direction: column;
+		}
+
+		.design-options {
+			display: flex;
+			gap: 2rem;
+		}
+	</style>
+	<h3>TabContainer Design Background</h3>
+	<div class="snippet">
+		<div class="design-options">
+			<div role="radiogroup" id="radioGroup-headerDesign" class="radio-button-group">
+				<ui5-title level="H4">Header Background</ui5-title>
+				<ui5-radio-button text="Solid" checked value-state="None" name="Group1"></ui5-radio-button>
+				<ui5-radio-button text="Transparent" value-state="None" name="Group1"></ui5-radio-button>
+			</div>
+
+			<div role="radiogroup2" id="radioGroup-contentDesign" class="radio-button-group">
+				<ui5-title level="H4">Content Background</ui5-title>
+				<ui5-radio-button text="Solid" value-state="None" name="Group2"></ui5-radio-button>
+				<ui5-radio-button text="Transparent" checked value-state="None" name="Group2"></ui5-radio-button>
+			</div>
+		</div>
+		<ui5-tabcontainer id="tabContainerBackgroundDesign" header-background-design="Solid" content-background-design="Transparent">
+			<ui5-tab text="Tab 1" selected>
+				<ui5-label>Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita voluptates accusamus est quibusdam inventore ipsam?</ui5-label>
+			</ui5-tab>
+			<ui5-tab text="Tab 2">
+				<ui5-label>Accusamus minus aperiam sunt ipsam eos quos et maxime facilis tempora neque ratione nemo laborum expedita aliquid officiis nisi, necessitatibus quasi distinctio?</ui5-label>
+			</ui5-tab>
+			<ui5-tab text="Tab 3">
+				<ui5-label>Dolores totam perferendis numquam incidunt obcaecati, id quo at alias rem deserunt praesentium repellat ipsum commodi consequuntur veniam et ducimus animi qui nobis accusantium tenetur eveniet culpa non!</ui5-label>
+			</ui5-tab>
+		</ui5-tabcontainer>
+
+		<script>
+			(function () {
```

Comment:
i don't think it's necessary for this to be an IIFE

Reply:
In AvatarGroup's sample page, all samples are done with an IIFE, which is why I decided to do so as well. Because I can't test it (ruby problem), I feel it's safer to leave it like this just in case.

Reply:
Every other sample page doesn't have it and it works there. You don't need the scoping of variables it provides. It would be confusing to the reader and might lead them to believe that an IIFE is required to use ui5-webcomponents.

Reply:
ok, will remove it